packages/inference/src/types.ts

```diff
@@ -51,6 +51,7 @@ export const INFERENCE_PROVIDERS = [
 	"replicate",
 	"sambanova",
 	"together",
+	"blackforestlabs-ai",
```

Comment:
```suggestion
	"black-forest-labs",
```

I think i would use the same id as your org (https://huggingface.co/black-forest-labs) wdyt?